kv/db.go

```diff
@@ -163,6 +165,71 @@ func (s *rpcDBServer) executeCmd(args proto.Request, reply proto.Response) error
 	return nil
 }
 
+func (s *rpcDBServer) CreateTable(args *proto.CreateTableRequest, reply *proto.CreateTableResponse) error {
+	schema := args.Schema
+	tableDesc := proto.TableDescriptor{
+		Id:    0, // allocate here
+		Table: schema.Table,
+	}
+
+	var columnsByName map[string]uint32
+
+	for _, column := range schema.Columns {
+		columnDesc := proto.ColumnDescriptor{
+			Id:     tableDesc.NextColumnId,
+			Column: column,
+		}
+
+		columnsByName[column.Name] = columnDesc.Id
```

Comment:
Should probably do some validation that column names are only specified once.

Should probably normalize column names to lower case.